eng/release/Scripts/GetBuildVersion.ps1

```diff
@@ -0,0 +1,38 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)][string] $BarId,
+    [Parameter(Mandatory=$true)][string] $MaestroToken,
+    [Parameter(Mandatory=$false)][string] $MaestroApiEndPoint = 'https://maestro-prod.westus2.cloudapp.azure.com',
```

Comment:
Is this always stable or should we be getting it from an environment variable or variable group?

Reply:
This is fairly stable. It's parameterized for easier update if it starts failing all of a sudden.